sdks/go/pkg/beam/core/runtime/harness/init/init.go

```diff
@@ -91,6 +100,13 @@ func hook() {
 			os.Exit(1)
 		}
 		runtime.GlobalOptions.Import(opt.Options)
+		var experiments []string
+		if e, ok := opt.Options.Options["experiments"]; ok {
+			experiments = strings.Split(e, ",")
+		}
+		if slices.Contains(experiments, "enable_data_sampling") {
+			runnerCapabilities = append(runnerCapabilities, graphx.URNDataSampling)
+		}
```

Comment:
Can you please add a TODO here to remove once the data sampling URN is properly sent in via the capabilities?

Reply:
Done : )